editor/dependency_editor.cpp

```diff
@@ -798,7 +800,12 @@ bool OrphanResourcesDialog::_fill_owners(EditorFileSystemDirectory *efsd, HashMa
 			if (!refs.has(path)) {
 				TreeItem *ti = files->create_item(p_parent);
 				ti->set_cell_mode(0, TreeItem::CELL_MODE_CHECK);
-				ti->set_text(0, efsd->get_file(i));
+				String additional_info;
+				if (scene_name == efsd->get_file(i)) {
```

Comment:
This will only match the file part no? Not the full path, does this really work with a main scene like `res://scenes/scene.tscn`

Reply:
Yes you are right, thanks! I now updated it to check the filename including the path.